doc/components_2.md

```diff
@@ -24,8 +24,6 @@ tags to make it easier to preview outside the main template
 </mjml>
 ```
 
-The `MJML` engine will then replace your included files before starting the rendering process
-
 <aside class="notice">
-Note that the file must be a file with a `.mjml` extension
+The file must have an '.mjml' extension.
```

Comment:
Whole line can be removed as it's no longer the case with current mj-include implem

Reply:
Yeah current mj include can include any file don't think we have to describe every behavior, we should describe uncommon cases let's remove the whole aside here

Reply:
Gone.